### DISCUSSION TITLE
[Feature] Locked Albums / 2fa

### DISCUSSION THREAD

**Discussion by ghostlyyll** [immich-app/immich#13372] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Ideally you would be able to have a 2fa to enter your account and etc. I would also like to see immich develop locked album that would offer 2x the security such as 2fa or maybe a security key as well

### Platform

- [x] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Please read the template before posting a feature request: this is multiple feature requests in one, _and_ they're both duplicates.
